src/current/v24.3/troubleshoot-replication-zones.md

```diff
@@ -0,0 +1,219 @@
+                |     range_min_bytes = 134217728,
+                |     range_max_bytes = 536870912,
+                |     gc.ttlseconds = 14400,
+                |     num_replicas = 5,
+                |     num_voters = 3,
+                |     constraints = '{+region=europe-west1: 1, +region=us-east1: 1, +region=us-west1: 1}',
+                |     voter_constraints = '[+region=us-east1]',
+                |     lease_preferences = '[[+region=us-east1]]'
+(1 row)
+~~~
+
+In this case, the output is expected for a multi-region cluster spread across 9 nodes that was configured using [Multi-region SQL]({% link {{ page.version.version }}/multiregion-overview.md %}). Since nothing has been manually modified, the output shows that `movr.users` is using the zone configuration from its parent `movr` database. For more information about how this works, see [How zone config inheritance works]({% link {{ page.version.version }}/configure-replication-zones.md %}#how-zone-config-inheritance-works).
+
+However, if the zone configuration had been manually modified, there could be inconsistencies in the output that would show a misconfiguration.
+
+For example:
+
+- If the [type of problem](#types-of-problems) were a [constraint violation](#monitoring), you'd want to check that the values in [`constraints`]({% link {{ page.version.version }}/configure-replication-zones.md %}#constraints), [`voter_constraints`]({% link {{ page.version.version }}/configure-replication-zones.md %}#voter_constraints), and [`lease_preferences`]({% link {{ page.version.version }}/configure-replication-zones.md %}#lease_preferences) were not logically inconsistent, which would cause constraint violations.
+- If the [type of problem](#types-of-problems) were an [under-replicated range](#monitoring), you'd want to check the values of [`num_replicas`]({% link {{ page.version.version }}/configure-replication-zones.md %}#num_replicas) and [`num_voters`]({% link {{ page.version.version }}/configure-replication-zones.md %}#num_voters). Modifying these values can cause under-replication.
+- If [range splits are failing]({% link {{ page.version.version }}/common-errors.md %}#split-failed-while-applying-backpressure-are-rows-updated-in-a-tight-loop), the value of [`gc.ttlseconds`]({% link {{ page.version.version }}/configure-replication-zones.md %}#gc-ttlseconds) may be too high.
+
+If the zone configuration for the target schema object looks good, [move to the next step](#step2).
+
+If the zone configuration does not look right, repair it now using [`ALTER TABLE ... CONFIGURE ZONE`]({% link {{ page.version.version }}/alter-table.md %}#configure-zone). You can set the field to another value, but often the best thing to do is to discard the changed settings using [`ALTER TABLE ... CONFIGURE ZONE DISCARD`]({% link {{ page.version.version }}/alter-table.md %}#remove-a-replication-zone) so that it returns to inheriting values from its parent object:
```

Comment:
```suggestion
If the zone configuration does not look right, repair it now using [`ALTER TABLE ... CONFIGURE ZONE`]({% link {{ page.version.version }}/alter-table.md %}#configure-zone). You can set the problem-causing field to another value, but often the best thing to do is to discard the changed settings using [`ALTER TABLE ... CONFIGURE ZONE DISCARD`]({% link {{ page.version.version }}/alter-table.md %}#remove-a-replication-zone) so that it returns to inheriting values from its parent object:
```